src/lib/builtins/string.rs

```diff
@@ -741,7 +741,9 @@ pub fn create_constructor(global: &Value) -> Value {
 
     // Create prototype
     let proto = ValueData::new_obj(Some(global));
-    let prop = Property::default().get(to_value(get_string_length as NativeFunctionData));
+    let prop = Property::default()
+        .get(to_value(get_string_length as NativeFunctionData))
```

Comment:
`String.prototype.length` does not have a getter according to the spec https://tc39.es/ecma262/#sec-properties-of-the-string-prototype-object